nautobot_data_validation_engine/__init__.py

```diff
@@ -26,5 +30,26 @@ class NautobotDataValidationEngineConfig(PluginConfig):
     default_settings = {}
     caching_config = {}
 
+    def ready(self):
+        super().ready()
+        from nautobot_data_validation_engine.template_content import tab_factory  # pylint: disable=C0415
+        from django.contrib.contenttypes.models import ContentType  # pylint: disable=C0415
+
+        template_content = []
+        labels = []
+        try:
+            for content_type in ContentType.objects.all():
```

Comment:
I think we should avoid this pattern, See here: https://github.com/nautobot/nautobot/pull/3633

Reply:
> Which part are you thinking we should avoid?

You should not need to make a call to the database to bring up the application. This may cause issues when that is forked into different workers as an example

Reply:
Gotcha.  Removed that piece of code and it's now pulling from the registry instead.